files/data/sbox_gra.mk

```diff
@@ -5,8 +5,12 @@ SBOX_GRA_FILES := \
 	sbox_gra_00000000.NCLR \
 	sbox_gra_00000001.NCGR
 
+4BPP_NOPAD_PCMP_NCLR_FILES += $(SBOX_GRA_DIR)/sbox_gra_00000000.NCLR
 VERSION101_SOPC_NCGR_FILES += $(SBOX_GRA_DIR)/sbox_gra_00000001.NCGR
 
+$(SBOX_GRA_DIR)/sbox_gra_00000000.NCLR: $(SBOX_GRA_DIR)/sbox_gra_00000001.png
+	$(GFX) $< $@ $(GFX_FLAGS)
```

Comment:
This should already be handled in ``graphics_files_rule.mk``

Reply:
In this case the stems do not match. `graphics_files_rules.mk` only handles the cases where the stems *do* match.